### DISCUSSION TITLE
Student benefits

### DISCUSSION THREAD

**Discussion by harisusan152** [community/community#137449] [GitHub Education]
### Hi everyone,

Hi everyone, I have a question. Approximately 4 days ago I applied for The student education pack and they approved my application and they said to wait for 3 days and now it has already past the 3 days time what to do?

**Comment by andrefqms** [comment]
The GitHub Student Developer Pack approval process typically takes 2-5 business days. Students must apply and provide proof of current student status, such as an active .edu email address or student ID.
But if it takes too long you should have email them regarding this issue.

**_https://support.github.com/contact/education_**

**Comment by davevad93** [comment]
Hi @harisusan152 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @harisusan152 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by KaveeshYoshitha** [comment]
Log into git hub student developer pack .
